pages/docs.json

```diff
@@ -105,6 +105,7 @@
       "sections": [
         { "title": "Can I nest rules?" },
         { "title": "Can I refer to other components?" },
+        { "title": "What is the difference between Comp.extend and styled(Comp)?" },
```

Comment:
Can we shorten this, @mxstbr?

This should maybe just be `When should I use styled()?`

Reply:
What about this: `Should I use Comp.extend or styled()?`
or maybe `Should I use extend or styled()?`